web/src/features/charts/FuturePrice.tsx

```diff
@@ -263,16 +323,35 @@ const getColor = (
   date: string,
   granularity: number
 ): string => {
-  if (price === maxPrice) {
-    return 'bg-danger dark:bg-red-400';
-  } else if (price === minPrice) {
-    return 'bg-success dark:bg-emerald-500';
-  } else if (
+  if (
     normalizeToGranularity(new Date(date), granularity) <
     normalizeToGranularity(new Date(), granularity)
   ) {
     return 'bg-[#18214F] dark:bg-[#848EC0] opacity-50';
+  } else if (
+    priceIn5Percentile(price, maxPrice, minPrice, true) &&
+    maxPrice != minPrice
+  ) {
+    return 'bg-danger dark:bg-red-400';
+  } else if (
+    priceIn5Percentile(price, maxPrice, minPrice, false) &&
+    maxPrice != minPrice
+  ) {
+    return 'bg-success dark:bg-emerald-500';
```

Comment:
Should the dark not be bg-success-dark?

@Alportan

Reply:
bg-emerald-500 is the same as bg-success-dark, except that it doesn't work bg-success-dark doesn't behave as expected. That's why i used bg-emerald-500, but maybe i should just fix it in the tailwind config instead

Reply:
> bg-emerald-500 is the same as bg-success-dark, except that it doesn't behave as expected. That's why i used bg-emerald-500, but maybe i should just fix it in the tailwind config instead

In what way does it not behave as expected? 👀

Reply:
It uses the bg-succes color

Reply:
`bg-success-dark` is indeed mapped to `emerald-500` in Figma, but it wasn't showing correctly if I recall correctly (I think @silkeholmebonnen did mentioned this to me a while back). If there's a way to make it work, it would be nice, otherwise we can also revise it later, maybe together with most colors. 

We could do another pair-code/design session, similar to the one for typography 🙌

Reply:
> `bg-success-dark` is indeed mapped to `emerald-500` in Figma, but it wasn't showing correctly if I recall correctly (I think @silkeholmebonnen did mentioned this to me a while back). If there's a way to make it work, it would be nice, otherwise we can also revise it later, maybe together with all colors to make sure the repo and Figma match.
> 
> We could do another pair-code/design session, similar to the one for typography 🙌

Reply:
Yes please 🙏

Reply:
Slack week topic, but we're mid way haha 😅

Reply:
Okay seems to work for me as well, idk what happened before. I'll change it to use `dark:bg-success-dark`